cl/search/models.py

```diff
@@ -2846,6 +2846,30 @@ class Meta:
         unique_together = ("citing_opinion", "cited_opinion")
 
 
+class Parenthetical(models.Model):
+    describing_opinion = models.ForeignKey(
+        Opinion,
+        related_name="authored_parentheticals",
+        on_delete=models.CASCADE,
+    )
+    described_opinion = models.ForeignKey(
+        Opinion, related_name="parentheticals", on_delete=models.CASCADE
+    )
+    text = models.TextField(
+        help_text="The text of the description as written in the describing opinion",
+    )
+    usefulness_score = models.FloatField(
+        db_index=True,
+        help_text="A score between 0 and 1 representing how descriptive the parenthetical is",
+    )
```

Comment:
We'll need this to have a default or be nullable, won't we?

Reply:
I think I would also just call this `score`, to keep it simple. That matches nicely with the simplicity of `text` too.

Reply:
Just need this to have a default or be nullable and I think we're there.

Reply:
Done